scripts/create-database.js

```diff
@@ -0,0 +1,54 @@
+const mysql = require('mysql2/promise');
```

Comment:
Comments like this are not really helpful, they just add noise to the file. Comments should really be used to explain _why_ something is being done the way it is and not _how_ (ideally, the code itself should be clear enough to not need any further explanation).

The indentation is also out, and there are blank spaces at the end of several of the lines. Just run it through the linter and all these problems are fixed for you!